assets/app/views/modals/debug-terminal.html

```diff
@@ -34,7 +34,15 @@ <h2 ng-if="debugPod.status.phase !== 'Failed'" class="text-muted">
       </div>
       <div ng-if="container | entrypoint : image" class="original-cmd-msg">
         <label>Original Command:</label>
-        <code>{{container | entrypoint : image}}</code>
+        <code>
+          <truncate-long-text
+              content="container | entrypoint : image"
+              limit="80"
+              newline-limit="1"
+              expandable="false"
```

Comment:
agreed there, stupid copy/paste

On Thu, Apr 14, 2016 at 2:50 PM, Sam Padgett <notifications@github.com>
wrote:

> In assets/app/views/modals/debug-terminal.html
> <https://github.com/jwforres/origin/pull/51#discussion_r59771082>:
>
> > @@ -34,7 +34,15 @@ <h2 ng-if="debugPod.status.phase !== 'Failed'" class="text-muted">
> >        </div>
> >        <div ng-if="container | entrypoint : image" class="original-cmd-msg">
> >          <label>Original Command:</label>
> > -        <code>{{container | entrypoint : image}}</code>
> > +        <code>
> > +          <truncate-long-text
> > +              content="container | entrypoint : image"
> > +              limit="80"
> > +              newline-limit="1"
> > +              expandable="false"
>
> I think a copy button would be more valuable, but we have the term.js
> paste problem.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/jwforres/origin/pull/51/files/55ea8e65cf4ae1390645c204bafaaf3094b65bc0#r59771082>
>